prediction_market_agent/agents/microchain_agent/app.py

```diff
@@ -96,7 +101,7 @@ def get_market_function_bullet_point_list() -> str:
     if not agent_is_initialized():
         model = st.selectbox(
             "Model",
-            ["gpt-4-turbo-2024-04-09", "gpt-3.5-turbo-0125"],
+            ["gpt-3.5-turbo-0125", "gpt-4o-2024-05-13"],
```

Comment:
default to gpt3.5. We're mostly still just experimenting, so it's preferable to run a cheaper model for most runs. Saves on fewer clicks. Also, switch gpt4 option to new `4o`.